src/pages/all-products/AllProducts.tsx

```diff
@@ -1,13 +1,21 @@
-import { productList } from '@common/constants';
-import { ProductListItem } from '@components/product-list-item/ProductListItem';
+import { Product } from '@common/type';
+import ProductList from '@components/product-list/ProductList';
+import ProductListTitle from '@components/product-list/ProductListTitle';
 import { Box } from '@mui/material';
+import { useGetProductsQuery } from '@services/ecommerce';
+import { createContext } from 'react';
+
+export const ProductContext = createContext<Product[] | undefined>(undefined);
 
 export default function AllProducts() {
+  const { data } = useGetProductsQuery(null);
```

Comment:
rename data, nobody knows what "data" it is. {data: allProducts}

Reply:
<img width="475" alt="Screenshot 2022-10-12 at 18 32 21" src="https://user-images.githubusercontent.com/64188262/195385728-069c87de-fa27-4c4e-a4c8-8433e2e68c3b.png">
do you need to pass null to useGetProductsQuery(null)? if you do not use it inside query: () =>

Reply:
or if I don't pass anything 
![image](https://user-images.githubusercontent.com/26207350/195387805-57fedf46-91c9-4d91-a07e-b7943bbefa82.png)